tests/decoding_test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+python tests/decoding_test.py
```

Comment:
This file is here because using !# does not work when using virtual environments and the toml-test does not support multiple arguments